manifests/server/grant.pp

```diff
@@ -323,7 +393,7 @@
   }
 
   $grant_cmd = "GRANT ${_privilege} ON ${_object_type} \"${_togrant_object}\" TO
-      \"${role}\""
+      ${role}"
```

Comment:
is it safe to do this? can the role have spaces?

Reply:
This is explicit to support `group <groupname>` in the role parameter. It seems Redshift explicitly supports spaces in the role name for this purpose, as can be seen in the aclitem[] example here: https://docs.aws.amazon.com/redshift/latest/dg/r_PG_DEFAULT_ACL.html

So, yes, spaces are supported. And while I'm nervous about removing string quoting here, it's required to support mixed use of including the group keyword, unless we opt to provide that as a separate variable. Quotation of the username or group is behavior that is, furthermore, not specified in https://docs.aws.amazon.com/redshift/latest/dg/r_GRANT.html

tl;dr, this is explicit, and since we're evaluating local code, should be safe. I can add a warning or tokenize this for safety if we can define scenarios in which misuse or injection (by user input, as opposed to puppet logic) will exist.

Reply:
I wasn't so worried about the security implications, mostly whether or not it would be considered valid syntax to say `GRANT my_privilege ON my_object_type my_object TO my cool role with spaces`. If your testing shows this is valid, then ship it!